easy_phi_client.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import json
+import urllib
+import logging
+
+import settings
+
+
+class PythonClient(object):
+    baseURL = settings.base_url
+
+    def __init__(self, api_token):
+        self.logger = logging.getLogger(__name__)
+        self.api_token = api_token
+
+    def _request(self, api_func, params=None):
```

Comment:
- missing docstrings
- this method should return Python object, not json
- this method should accept POST data as an optional paramenter